packages/amplify_flutter/lib/amplify.dart

```diff
@@ -120,4 +122,13 @@ class AmplifyClass extends PlatformInterface {
     PlatformInterface.verifyToken(instance, _token);
     _instance = instance;
   }
+
+  bool get isConfigured {
+    return _isConfigured;
+  }
 }
+
+enum HubChannel {
+  Auth,
+  DataStore
+}
```

Comment:
put it in a new file and export from here?

Reply:
Done.

Reply:
Done.